app/scripts/modules/amazon/pipeline/stages/cloneServerGroup/awsCloneServerGroupStage.js

```diff
@@ -55,8 +55,10 @@ module.exports = angular.module('spinnaker.core.pipeline.stage.aws.cloneServerGr
       stage.credentials = $scope.application.defaultCredentials.aws;
     }
 
-    if (stage.isNew && _.get($scope, 'application.attributes.providerSettings.aws.useAmiBlockDeviceMappings')) {
-      stage.useAmiBlockDeviceMappings = $scope.application.attributes.providerSettings.aws.useAmiBlockDeviceMappings;
+    if (stage.isNew) {
+      let useAmiBlockDeviceMappings = $scope.application.attributes.providerSettings.aws.useAmiBlockDeviceMappings || false;
```

Comment:
you want to wrap this is a `_.get` so it won't blow up if someone doesn't have that field set in their application, which is fairly likely, e.g.
```javascript
const useAmiBlockDeviceMappings = _.get($scope, 'application.attributes.providerSettings.aws.useAmiBlockDeviceMappings', false);
```
the third argument is the fallback in case the property doesn't exist